packages/esm-patient-registration-app/src/patient-registration/input/input.scss

```diff
@@ -88,8 +88,6 @@
 .IDInput {
   display: grid;
   grid-template-columns: 1fr auto;
-  align-items: center;
```

Comment:
Doesn't having `align-items: end` do the required work along with the fix to the button size. 

Please check and let me know.
Thanks!